tests/testthat/test-cut.R

```diff
@@ -0,0 +1,397 @@
+context("cut function")
+
+with_mock_crunch({
+    ds <- loadDataset("test ds")
+    gen <- ds$gender
+
+    test_that("cut labels are generated correcty", {
+        expect_identical(
+            crunch:::generateCutLabels(5, c(2.111111, 3, 4, 5), 4, FALSE, FALSE),
+            c("[2.1111,3)", "[3,4)", "[4,5)")
+        )
+        expect_identical(
+            generateCutLabels(2, c(2.111111, 3, 4, 5), 4, FALSE, FALSE),
+            c("[2.1,3)", "[3,4)", "[4,5)")
+        )
```

Comment:
You don't need `crunch:::` here, the way that testthat runs its tests it loads even unexported functions